src/main/java/nl/tudelft/ti2206/group9/server/HighscoreServer.java

```diff
@@ -35,16 +38,22 @@ private HighscoreServer() { }
      * @throws IOException when something unexpected happens.
      */
     public static void main(final String... args) throws IOException {
-        new Thread(new CLIThread(), "CLIThread").start();
-        try (ServerSocket serverSocket = new ServerSocket(PORT)) {
-            ss = serverSocket;
+        cliThread = new CLIThread();
+        new Thread(cliThread, "CLIThread").start();
+        log("Type \"stop\" or \"q\" to exit.");
+        try (ServerSocket sock = new ServerSocket(PORT)) {
+            serverSocket = sock;
             log("Server is now accepting clients.");
+            Socket socket;
```

Comment:
I would say that variables being declared inside loops is not good performance-wise. But, as the loop blocks until a client connects, it can be moved inside. Will do!

<review, logic